kuma/wiki/templates/wiki/document.html

```diff
@@ -390,7 +391,10 @@ <h2 class="offscreen">{{ _('Document Tags and Contributors') }}</h2>
                 </div>
               {% endif %}
             </div>
-
+	    {% if is_admin %}
+	    <center>Document number : {{ document.id }}</center>
```

Comment:
Let's try to match the existing styling for this kind of page information, we can borrow the same HTML structure too. 

The document number can be inserted right after the "last updated by" information. You can also copy how that information is being marked up and just replace the relevant parts: https://github.com/mozilla/kuma/blob/6d302a3a98cca6bd35e51e2e458417668c0d7eb0/kuma/wiki/templates/wiki/document.html#L377-L383

Don't copy the icon part, since we don't have an icon for this.